scripts/production/run_healthchecks.pl

```diff
@@ -0,0 +1,211 @@
+    $ensj_testrunner = $ENV{ENSEMBL_CVS_ROOT_DIR} . '/ensj-healthcheck/run-configurable-testrunner.sh';
+}
+die "'$ensj_testrunner' is not a valid executable" unless -x $ensj_testrunner;
+
+unless ($ensj_json_config) {
+    die "Need to give the --ensj-config option or set the ENSEMBL_CVS_ROOT_DIR environment variable to use the default" unless $ENV{ENSEMBL_CVS_ROOT_DIR};
+    $ensj_json_config = $ENV{ENSEMBL_CVS_ROOT_DIR} . "/ensembl-compara/conf/$division/ensj-healthcheck.json";
+}
+die "'$ensj_json_config' is not a valid file" unless -e $ensj_json_config;
+my $ensj_config = decode_json(slurp($ensj_json_config));
+
+
+# Common parameters
+my @params = (
+    '--host'    => $dba->dbc->host,
+    '--port'    => $dba->dbc->port,
+    '--driver'  => 'org.gjt.mm.mysql.Driver',
+    '--release' => $ENV{CURR_ENSEMBL_RELEASE} || software_version(),
+    '--test_databases'  => $dba->dbc->dbname,
+);
+
+# RO or RW user depending on the --repair option
+if ($repair) {
+    push @params, (
+        '--user'    =>  Bio::EnsEMBL::Compara::Utils::Registry::get_rw_user($dba->dbc->host),
+        '--password'=>  Bio::EnsEMBL::Compara::Utils::Registry::get_rw_pass($dba->dbc->host),
+        '--repair'  => 1,
+    );
+} else {
+    push @params, (
+        '--user'    =>  'ensro',
+    );
+}
+
+# Add the master database if one is available (otherwise defaults to ensj-healthcheck name matching)
+if ($master_db) {
+    my $master_dba = Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->go_figure_compara_dba($master_db);
+    die "Cannot find a database with the alias '$master_db' in the Registry" unless $master_dba;
+    push @params, (
+        '--compara_master.database', $master_dba->dbc->dbname,
+    );
+}
+
+# Division-specific configuration
+foreach my $key (qw(host1 host2 host3 secondary.host)) {
+
+    # Extract the prefix and the suffix to form the other arguments
+    my ($key_prefix, $key_suffix) = split('host', $key);
+
+    # Configure the host if it is set
+    if (my $host = $ensj_config->{$key}) {
+        my $port = Bio::EnsEMBL::Compara::Utils::Registry::get_port($host);
+        push @params, (
+            "--${key_prefix}host${key_suffix}"    => $host,
+            "--${key_prefix}port${key_suffix}"    => $port,
+            "--${key_prefix}user${key_suffix}"    => 'ensro',
+            "--${key_prefix}driver${key_suffix}"  => 'org.gjt.mm.mysql.Driver',
+        );
+    } else {
+        # Trick to tell the HC to ignore the default value that may be set in database.defaults.properties
+        # We rely on the fact that the HC doesn't die if the host name is not valid
+        push @params, (
+            "--${key_prefix}host${key_suffix}"    => '""',
+        );
+    }
+}
+
+print "Executing: ", join(" ", $ensj_testrunner, @params, @ARGV), "\n\n";
+
+# Need to change directory because database.default.properties is read from the current directory
+chdir dirname($ensj_testrunner);
+exec($ensj_testrunner, @params, @ARGV);
```

Comment:
Does this leave you in the same path where you called the script after if finishes?

Reply:
Yes, the `chdir` only applies to the script's process, and does no bubble up to your shell